Split webpack bundle in vendor and local

ffissore opened the pull request:
Split webpack bundle in `vendor` and `local`, use HashedModuleIdsPlugin so that the hash reflects each file contents. Vendor bundle is made of our dependencies, so it's unlikely to change: this way browsers can cache it more easily

jviotti reviewed (approved):
Looks good code wise, but I might not know the implications in detail

ffissore committed:
UI: Split webpack bundle in vendor and local, use
HashedModuleIdsPlugin so that the hash reflects each file contents.
Vendor bundle is made of our dependencies, so it's unlikely to
change: this way browser can cache it more easily

Change-type: patch
Signed-off-by: Federico Fissore <federico@balena.io>
